2-mandatory/8-password-validator.js

```diff
@@ -22,8 +22,20 @@ Expected Result:
 PasswordValidationResult=  [false, false, false, false, true]
 
 */
+validatePasswords(["StUFf27%", "Pl3nty!", "Jai33", "shajsaUA**&&", "Pl3nty!"]);
+const validatePasswords = (passwordArray) => passwordArray.map(isValidPassword);
 
-function validatePasswords(passwords) {}
+const isValidPassword = (password, index, array) =>
```

Comment:
As this function is quite a few lines, I would encourage you to wrap them in the curly braces / `{ }`. This is just a styling preference, but it makes things much easier to read and easy to understand where the function starts and stops :) 

I would say if a function is over 2 or 3 lines long, you should wrap them in curly braces and use the `return` keyword at the end. 🙌